srcpkgs/conky/template

```diff
@@ -19,11 +19,16 @@ license="BSD-3-Clause, GPL-3.0-or-later"
 homepage="https://github.com/brndnmtthws/conky"
 distfiles="https://github.com/brndnmtthws/conky/archive/v${version}.tar.gz
  https://github.com/brndnmtthws/conky/releases/download/v${version}/conky.1.gz"
-checksum="b0cd6a9197de1db527953f24635b4e4f19d8ab1258854c4adbfdd6e0f1588341
- 41c14d80a5ee6074ddcb1c3f05805f3f3bb0e65d13c164e619dd7ecbbba314eb"
+checksum="8633b78e6c0c9e7128efc9fe54b48df75a3860928e3fb101bcf71f6fb3844959
+ 4e2f31b09deb60f289f74d9e7f75e2e987ee0c5d400fc4cbf4ef2b60fe457295"
 
 post_extract() {
 	mv conky-*/* .
+	case "${XBPS_TARGET_MACHINE}" in
```

Comment:
selecting on target machine is wrong, check if cross instead

Reply:
that's what i did the first time. did i f it up?

![fail](https://github.com/user-attachments/assets/24edc83a-5d47-4289-a30c-1d4ea2bddf35)

Reply:
yes, yes i did, `!`